### PR TITLE
Fixed Issue #33

### PR DESCRIPTION
**Pull request by ziegenludwig**
Changed concatenating print statements to f-strings to improve reading ability and performance.

### CODE REVIEW

**Comment by LaiAlexander** [comment]
Hi!
Thank you for your contribution!

This is still missing some strings that may be converted to f-strings.
Some examples (this is not a complete list)
https://github.com/ESN-Trondheim/ESNbot/blob/3fdf822eb479f5dad88b9255257d9c24e942f6a7/esnbot/commands.py#L57

https://github.com/ESN-Trondheim/ESNbot/blob/3fdf822eb479f5dad88b9255257d9c24e942f6a7/esnbot/commands.py#L60-L61

https://github.com/ESN-Trondheim/ESNbot/blob/3fdf822eb479f5dad88b9255257d9c24e942f6a7/esnbot/main.py#L31-L32

**Comment by ziegenludwig** [comment]
I think I should of gotten most if not all now.

File: esnbot/commands.py [hunk 1]
```diff
@@ -70,27 +68,25 @@ def vinstraff(client, channel, user, argument, output):
...
+        client.respond_to(channel, user, f"Sorry, could not find anyone named '{argument[0]}'")
 
 def watermark(client, channel, user, argument, output):
     #if output.get('subtype') != "file_share":
     #    # command_help expects an array containing the help item
     #    # Displays help for watermark if watermark is not called from a file upload
     #    command_help(channel, ["watermark"], user, output)
     #else:
-    log_to_console("Arguments supplied by user: " + str(argument))
+    log_to_console(f"Arguments supplied by user: {str(argument)}")
```
Comment (LaiAlexander) [inline esnbot/commands.py:89]: There's not really a point in casting this to str, the f-string already does this anyway.

File: esnbot/commands.py [hunk 2]
```diff
@@ -154,7 +150,7 @@ def watermark(client, channel, user, argument, output):
     log_to_console("File deleted from system...")
         
 def coverphoto(client, channel, user, argument, output):
-    log_to_console("Arguments supplied by user: " + str(argument))
+    log_to_console(f"Arguments supplied by user: {str(argument)}")
```
Comment (LaiAlexander) [inline esnbot/commands.py:153]: There's not really a point in casting this to str, the f-string already does this anyway.

File: esnbot/main.py [hunk 3]
```diff
@@ -28,8 +28,8 @@ def run(self):
           Main function
       """
       if self.slack_client.rtm_connect(auto_reconnect=True, with_team_state=False):
-          log_to_file_and_console("log" + os.sep + "connected.log",
-                                  "ESNbot connected and running...", "w")
+          log_to_file_and_console(f"log {os.sep} connected.log,
+                                  ESNbot connected and running..., w")
```
Comment (LaiAlexander) [inline esnbot/main.py:32]: `def log_to_file_and_console(filename, message, mode)`

This should take three arguments, not one.

**Review by LaiAlexander — CHANGES_REQUESTED** [review]
Hi again! Apologies for the late reply. 
Please see the comments, and fix those issues. 
Also, there's been quite some changes since you originally submitted this PR. Please merge master into your branch and resolve the merge conflicts.
We have also recently added [contributing guidelines](https://github.com/ESN-Trondheim/ESNbot/blob/master/CONTRIBUTING.md). Please read them and follow the steps, particularly the already mentioned merge from master into your branch, formatting and that all checks needs to pass for a PR to be accepted.

**Pull request closed without merging**